README-en.md

```diff
@@ -0,0 +1,382 @@
+
+#### Technical Specification
+The following is CPU core's specification.
+
+| Item | Value |
+---- | ----
+Operational voltage | 1.7 to 3.6 V |
+| Maximum RF Power | 4dBm |
+| Flash size | 512KB |
+| RAM size | 64KB |
+
+### Daughter board
+[daughterboard] (https://user-images.githubusercontent.com/135050/58088621-078b7380-7bfe-11e9-8d0d-8d9c216770b4.jpg)
+
+Mounted on the daughter board are LEDs, tactile switches, temperature sensors, and an OLED screen.
+**Please ensure the orientation of the CPU core when using the daughter board. Permanent damages may occur if placed incorrectly.**
+
+#### Technical Specification
+| Item | Value |
+---- | ----
+Power source | Battery / USB / external power supply |
+Operational voltage (battery) | 3V (2 single AAA batteries 1.5V in series) |
+Operational voltage (USB) | 5V |
+Operational voltage (external supply) | 5V |
+| Maximum continuous current consumption (battery powered / motor not used) | 0.6A |
+| Maximum continuous current consumption (when using motor) | 2.6A |
+
+## Peripherals on the daughter board
+**NOTE 5V systems and 3.3V systems cannnot be directly connected to one another.**
+The development board will act as a 5V system if powered by an external power source. Conversely, the development board will be a 3.3V system if powered by batteries. Please proceed with caution and make sure all connections are made with their corresponding voltages.
+**You WILL damage your board if connections to an incompatible voltage system is made.**
+
+### LED
+The anode of the LED is connected to the microcontroller. Therefore, when the microcontroller outputs High, the LED will turn on. Please set the port to output when using it.
+
+| Board number | Microcontroller pin number |
+---- | ----
+| DS2 | 7 |
+| DS3 | 11 |
+| DS4 | 19 |
+| DS5 | 17 |
+
+### Tactile switch
+The tactile switches are not pulled up nor down on the circuit, therefore, you will need to set the pins to INPUT_PULLUP when programming the microcontroller.
+
+| Board number | Microcontroller pin number |
+---- | ----
+| SW1 | 29 |
+| SW2 | 28 |
+
+### Buzzer
+A piezoelectric buzzer is included. To make the buzzer sound, you will need to provide an AC signal from the microcontroller. Setting the pin to HIGH will not cause the buzzer to make a sound. Normally on a regular Arduino, you would be able to use the tone() function to create such signals, however the timer in the nRF52832 is incompatible with this library. Therefore, you will need to generate the signal yourself through timers, interrupts, etc. In the sample provided, we take advantage of FreeRTOS's timer to generate a 1KHz interrupt to toggle the pin state which creates a 500Hz signal to make the buzzer sound. If you want to change the pitch of the buzzer, you will need to play around with the timers. 
+You can use the tone () function to produce any sound with regular Arduino, but you can not use this with the nRF52832 Arduino environment. 
```

Comment:
not need

Reply:
@cpulabs This sentence is from 

> 通常のArduinoであればtone()関数で任意の音を出すことが可能ですが、nRF52832のArduino環境ではこれを使うことができません。

Is this unnecessary?